charts/victoria-metrics-agent/templates/pdb.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "chart.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+  {{- if .Values.podDisruptionBudget.labels }}
```

Comment:
could you please use {{ with }}, i think this is more readable

Reply:
Sure i'll do that.

Reply:
Fixed